### PR TITLE
add wall-follow and docker instructions

### PR DESCRIPTION
**Pull request by alsora**
Signed-off-by: Alberto Soragna <alberto.soragna@gmail.com>

### CODE REVIEW

File: docs/api/wall-follow.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+The Create® 3 robot exposes a ROS 2 action server to invoke a wall-following behavior.
+
+You can command the robot to follow along an obstacle using the following ROS 2 action.
+
+```sh
+ros2 action send_goal /wall_follow irobot_create_msgs/action/WallFollow "{follow_side: 1, max_runtime: {sec: 1, nanosec: 0}}"
+```
+
+When this behavior is requested, the robot will try to engage with nearby obstacles and, after a successfull engagement, it will start following the obstacle along the specified side until the maximum runtime is reached.
```
Comment (shamlian) [inline docs/api/wall-follow.md:11]: successfullllllll

File: mkdocs.yml [hunk 2]
```diff
@@ -73,6 +74,7 @@ nav:
       - Reflexes: api/reflexes.md
       - Safety: api/safety.md
       - User Interface: api/ui.md
+      - Wall-Follow: api/wall-follow.md
```
Comment (shamlian) [inline mkdocs.yml:77]: I think it should be "Wall Follow" (no `-` -- this is an inconsistent usage)

**Review by shamlian — CHANGES_REQUESTED** [review]
Just a couple minor typos to fix, otherwise it looks good.

**Review by shamlian — CHANGES_REQUESTED** [review]
Just a couple minor typos to fix, otherwise it looks good.

**Review by shamlian — APPROVED** [review]

**Review by sebastianfer — APPROVED** [review]

**Review by justinIRBT — APPROVED** [review]

**Pull request merged**